README.md

```diff
@@ -14,7 +14,7 @@ See [PrestoDB: Mission and Architecture](ARCHITECTURE.md).
 ## Requirements
 
 * Mac OS X or Linux
-* Java 8 Update 151 or higher (8u151+), 64-bit. Both Oracle JDK and OpenJDK are supported.
+* Java 17 64-bit. Both Oracle JDK and OpenJDK are supported.
```

Comment:
Do we need to be very specific about the JDK version like "17.0.13" because of the `2024a` timezone issue Anant was talking about in our Slack?

Reply:
We will upgrade tzdata to 2025a very soon which would quickly make this out of date. In fact, if we recommended using 8u151, users would see lots of different behavior due to out of date tzdata. The only reason we've pinned to 17.0.13 is because tzdata 2024b/2025a introduced changes to time zones which Presto makes assumptions about in tests, causing some assertions to fail